units-validation/src/main/java/io/airlift/unitsvalidation/MaxDataSize.java

```diff
@@ -11,7 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.airlift.units;
+package io.airlift.unitsvalidation;
```

Comment:
It needs to not be on the list:
https://github.com/prestosql/presto/blob/8be3d79e34c1ea5df1a05ff64904e358f2be9a04/presto-main/src/main/java/io/prestosql/server/PluginManager.java#L65-L71